index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/e746991a9e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Ibute_uchenna-portfolio</title>
+</head>
+<body>
```

Comment:
- [ ] Your efforts are to salute @Uchenna-ibute, you have done great progress, but let's take a look at this image below, seems like your design is not well aligned and styled as per the Figma design, please modify the changes comparing these 2 images

> | correct image    | wrong image |
> |-----------------|--------------|
> | <img width="311" alt="im4" src="https://user-images.githubusercontent.com/77843050/204524294-d7b96e64-3fd8-4c9e-bcbe-7bd31b06cc0f.png"> | <img width="427" alt="im2" src="https://user-images.githubusercontent.com/77843050/204524391-72c46642-7cb1-4093-850f-36d221307a1e.png"> |

- [ ] We can see the presence of some borders of the design, referring to the Figma design provided by microverse, we do not have them, please take a look at it and fix it as you will be submitting the second review.
- [ ] Your text font style is a bit different from the one on the Figma design, please pick and stick to the same properties used in the figma, you are requested to use the same, `padding, margin, color, fonts...` as per the Figma design

> | correct image    | wrong image |
> |-----------------|--------------|
> | <img width="311" alt="im4" src="https://user-images.githubusercontent.com/77843050/204524294-d7b96e64-3fd8-4c9e-bcbe-7bd31b06cc0f.png"> | <img width="384" alt="im3" src="https://user-images.githubusercontent.com/77843050/204525837-017c6f7d-1001-4ca0-aa94-8e6ffe51dcc4.png"> |